src/citrine/resources/dataset.py

```diff
@@ -305,9 +328,9 @@ def delete_contents(
             deleted.
 
         """
-        path = format_escaped_url('projects/{project_id}/datasets/{dataset_uid}/contents',
+        path = format_escaped_url('teams/{team_id}/datasets/{dataset_uid}/contents',
                                   dataset_uid=self.uid,
-                                  project_id=self.project_id
+                                  team_id=self.team_id
```

Comment:
I see, even the `team_id` is not mandatory for the dataset's collection both the project and team initialize the collection with the corresponding `team_id`. This is ok.